.concourse/pipeline.yaml.gomplate

```diff
@@ -127,7 +127,9 @@ resources:
     access_key_id: ((aws-access-key))
     secret_access_key: ((aws-secret-key))
     region_name: {{ $config.s3_bucket_region }}
-    regexp: kubecf-v(.*).tgz
+    # adapted from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+    # with 1 capturing group for the whole semver, as the s3 resource expects
+    regexp: kubecf-[0-9]+\.[0-9]+\.[0-9]+(?:-[0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*)?(?:\+[0-9A-Za-z-]+)?.tgz
```

Comment:
As far as I can tell, there's no longer a capturing group at all (all groups are non-capturing, `(?:`).
```suggestion
    regexp: kubecf-([0-9]+\.[0-9]+\.[0-9]+(?:-[0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*)?(?:\+[0-9A-Za-z-]+)?).tgz
```

Reply:
Totally true, I managed to mangle my last interactive rebase. Pushed a new commit.